### ISSUE TITLE
`accessibilityState` of swipeable rows should be dynamically updated

### ISSUE THREAD

**Issue by sha-265** [ZeusLN/zeus#1752]
### Describe the bug

The `accessibilityState` attribute of the wallet's swipeable rows (part of the [`LayerBalances`](https://github.com/ZeusLN/zeus/blob/ebf5d0674dd996b0bd9e8c2553188bf8e5783de4/components/LayerBalances/index.tsx#L62) component) is not updated dynamically. 

For better accessibility the `accessibilityState` value should be update dynamically when swiping the rows, from `{ expanded: false }` to `{ expanded: true }` and vice versa.

As I understand maybe we need to use [direct manipulation](https://reactnative.dev/docs/0.70/direct-manipulation), or as @kaloudis [suggested](https://github.com/ZeusLN/zeus/pull/1590#issuecomment-1739452837) to track the state of the rows somehow.

### Reproduce

Just swipe the rows with screen reader enable, and listen to `accessibilityState` state.

### Zeus version

v0.8.0

### Node interface

Multiple (specified in comments)

### Network

Clearnet

### Device

Any

### Device operating system

Any

**Comment by kaloudis** [comment]
addressed in https://github.com/ZeusLN/zeus/pull/4433

**Comment by kaloudis** [comment]
ZEUS v13.2.1-rc1 which includes a fix for this, is now available for testing: https://github.com/ZeusLN/zeus/releases/tag/v13.2.1-rc1

It will be submitted to app stores shortly. Please reopen if you hit any problems.

Thank you for reporting.

**Comment by kaloudis** [comment]
@sha-265 ZEUS v13.2.1, which includes a fix, is now available on app stores https://github.com/ZeusLN/zeus/releases/tag/v13.2.1

Please reopen the issue and report back if it recurs. Thanks again for reporting.
